packages/button/package.json

```diff
@@ -21,9 +21,9 @@
   "keywords": [],
   "dependencies": {
     "@pluralsight/ps-design-system-core": "^4.4.2",
+    "@pluralsight/ps-design-system-filter-react-props": "^1.1.3",
     "@pluralsight/ps-design-system-icon": "^12.0.1",
     "@pluralsight/ps-design-system-theme": "^2.0.2",
-    "@pluralsight/ps-design-system-util": "^2.0.2",
```

Comment:
i think you still need this for the `transparentize` function

Reply:
I don't think I'm using transparentize any more.  Unless grep is failing me...

Reply:
Wait, you're right, this is `button`...